papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,423 @@
+   :figclass: w
+
+   The curves show the traces of the log posterior
+   value as well as traces of the values of the prior parameter values. The *SmoothingparamN* parameters
+   correspond to the smoothing parameters per multi-scale level. The solid horizontal orange lines show the mean
+   value, the shaded orange area the :math:`1~\sigma` error region. The burn in phase is shown transparent and ignored
+   while estimating the mean.  :label:`diagnosis1`
+
+
+*Pylira* relies on an MCMC sampling approach to sample a series of reconstructed images from the posterior
+likelihood defined by Eq. :ref:`post`. Along with the sampling it marginalises over the smoothing
+hyper-parameters and optimizes them in the same process. To diagnose the validity of the results it is
+important to visualise the sampling traces of both the sampled images as well as hyper-parameters.
+
+Fig. :ref:`diagnosis1` shows one typical diagnostics plot created by the code example above.
+In a multi-panel figure user can inspect the traces of the total log-posteriror as well as the
+traces of th smoothing parameters. Each panel corresponds smoothing hyper parameter
+introduced for each level of the multi-scale representation of the reconstructed image.
+The figure also shows the mean value along with the :math:`1~\sigma` error
+region. In this case the algorithm show stable convergence after a burn-in phase of approximately 200
+iterations for the log-posterior as well as all of the multi-scale smoothing parameters.
+
+
+.. figure:: images/pylira-diagnosis-pixel.pdf
+   :scale: 60%
+   :align: center
+   :figclass: w
+
+   The curves show the traces of value the pixel of interest for a simulated point source and its neighboring
+   pixels (see code example). The image on the left shows the posterior mean. The white circle in the image
+   shows the circular region defining the neighboring pixels. The blue line on the right plot shows the trace
+   of the pixel of interest. The solid horizontal orange lines show the mean value, the shaded orange area
+   the :math:`1~\sigma` error region. The burn in phase is shown in transparent blue and ignored while computing
+   the mean. The shaded gray lines show the traces of the neighboring pixels.  :label:`diagnosis2`
+
+
+Another useful diagnostic plot is shown in Fig. :ref:`diagnosis2`. The plot shows the
+image sampling trace for a single pixel of interest and its surrounding circular region of interest.
+This visualisation allows user to asses the stability of a small region in the image
+e.g. an astronomical point source during the MCMC sampling process. Due to the correlation with
+neighbouring pixels the actual value of a pixel might vary in the sampling process, which appears
+as "dips" in the trace of the pixel of interested and anti-correlated "peaks" in the one or mutiple
+of the surrounding pixels. In the this example a stable state of the pixels of interest
+is reached after approximately 1000 iterations.
+
+
```

Comment:
Perhaps you can include a description of the significance method for the additional structure and use the example from McKeough et al 2016 paper
https://ui.adsabs.harvard.edu/abs/2016ApJ...833..123M/abstract

I guess it would be good to get Karthik routines into the code and use it as an example.

Reply:
I agree this would be great! I'll still wait for Karthik's response...